### DISCUSSION TITLE
Snap to grid

### DISCUSSION THREAD

**Discussion by JulienHeijmans** [JulienHeijmans/quicksnap#24] [Potential Upcoming features]
Still unsure if that feature fits within the tool. I am also not a fan myself of snapping to grids, of how the active grid is chosen based on camera angle. It always feels finicky when the camera is not aligned with one of the axis.

But it has been requested twice already, so I am adding it here in the meantime.

**Comment by d1n3301** [comment]
Yes, a snap to the grid is very useful.
